apps/meteor/app/livechat/server/lib/Helper.ts

```diff
@@ -141,10 +140,7 @@ export const createLivechatRoom = async <
 	}
 
 	await callbacks.run('livechat.newRoom', room);
-
-	// TODO: replace with `Message.saveSystemMessage`
-
-	await sendMessage(guest, { t: 'livechat-started', msg: '', groupable: false, token: guest.token }, room);
+	await Message.saveSystemMessageAndNotifyUser('livechat-started', rid, '', { _id, username }, { groupable: false });
```

Comment:
looks like you removed `token` for some reason.. I'm suggesting removing `groupable` since `saveSystemMessageAndNotifyUser` already adds it
```suggestion
	await Message.saveSystemMessageAndNotifyUser('livechat-started', rid, '', { _id, username }, { token: guest.token });
```